src/material-experimental/mdc-slider/slider.ts

```diff
@@ -773,9 +772,53 @@ export class MatSlider extends _MatSliderMixinBase
 class SliderAdapter implements MDCSliderAdapter {
 
   /** The global change listener subscription used to handle change events on the slider inputs. */
-  changeSubscription: Subscription;
+  private _changeSubscription: Subscription;
```

Comment:
`private _changeSubscriptions = new Subscription()` so we guarantee no issues with undefined